runtime/lua/vim/treesitter.lua

```diff
@@ -348,6 +348,9 @@ function M.get_captures_at_cursor(winnr)
   return captures
 end
 
+--- WARNING: Will exhibit undefined behavior if the buffer has not been parsed up-to-date:
+--- `vim.treesitter.get_parser(opts.bufnr):parse(range)`
```

Comment:
That looks like copy and paste out of specific code (what is `opts` here?) -- should be made more generic.

Reply:
It referred to the `opts` param of `get_node`, but I suppose that wasn't clear, so I've removed it